Python/bytecodes.c

```diff
@@ -1158,31 +1169,43 @@ dummy_func(
             }
         }
 
-        inst(LOAD_NAME, ( -- v)) {
-            PyObject *name = GETITEM(frame->f_code->co_names, oparg);
-            PyObject *locals = LOCALS();
-            if (locals == NULL) {
+        op(_LOAD_NAME_INTRO, (-- mod_or_class_dict, name)) {
+            name = GETITEM(frame->f_code->co_names, oparg);
+            mod_or_class_dict = LOCALS();
+            if (mod_or_class_dict == NULL) {
```

Comment:
Aside: This should probably be an assert (no need to change it for this PR)

Reply:
Yes, the different LOAD_* instructions are a bit inconsistent about this. I can try to do some harmonization work here once we're on 3.13, but wanted to minimize any possible behavior changes on the existing instructions.